src/containerapp/azext_containerapp/_help.py

```diff
@@ -0,0 +1,533 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['containerapp'] = """
+    type: group
+    short-summary: Commands to manage Azure Container Apps.
```

Comment:
Change to match other top level CLI commands

```suggestion
    short-summary: Manage Azure Container Apps.
```